utils/cookies_login.py

```
@@ -0,0 +1,348 @@
+            }
+        return result
+
+async def interactive_login():
+    manager = PlatformLoginManager()
+    platforms = list(manager.platforms.items())
+    
+    while True:
+        print("\n" + "=" * 45)
+        print("🌟 N.E.K.O 安全凭证管理终端 (Security V2) 🌟")
+        print("=" * 45)
+        for i, (key, info) in enumerate(platforms, 1):
```

Ruff (0.15.1)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)